apps/researcher/src/messages/nl/messages.json

```diff
@@ -120,7 +120,16 @@
     "transferredTo": "Overgedragen aan",
     "location": "Locatie",
     "descriptionTitle": "Beschrijving",
-    "noData": "Geen herkomstgegevens gevonden."
+    "noData": "Geen herkomstgegevens gevonden.",
+    "dataTableTitle": "Herkomst data tabel",
+    "timelineTitle": "Tijdlijn van herkomst",
+    "timelineDescription": "In de tijdlijn worden de herkomst gebeurtenissen uitgezet op basis van de tijd. Klik op een gebeurtenis om de focus op de gebeurtenis te leggen.",
+    "timelineEventsWithoutDates": "{eventCount, plural, =1 {Herkomst gebeurtenis {events} is niet geplot op de tijdlijn omdat we geen data beschikbaar hebben.} other {# Herkomst gebeurtenissen {events} worden niet geplot op de tijdlijn omdat we geen data beschikbaar hebben}}",
```

Comment:
1. Minor: `herkomstgebeurtenis` and `herkomstgebeurtenissen` (as one word).
2. `omdat we geen data beschikbaar hebben` --> `omdat er geen data beschikbaar zijn`?